test/Core.Test/AdminConsole/Services/OrganizationServiceTests.cs

```diff
@@ -2362,6 +2339,15 @@ private void SetupOrgUserRepositoryCreateManyAsyncMock(IOrganizationUserReposito
                 return Task.FromResult<ICollection<Guid>>(orgUsers.Select(u => u.Id).ToList());
             }
         );
+
+        organizationUserRepository.CreateAsync(Arg.Any<OrganizationUser>(), Arg.Any<IEnumerable<CollectionAccessSelection>>()).Returns(
+            info =>
+            {
+                var orgUser = info.Arg<OrganizationUser>();
+                orgUser.Id = Guid.NewGuid();
+                return Task.FromResult<Guid>(orgUser.Id);
+            }
+        );
```

Comment:
This was a tricky one - the invite user method assumes that the repository will set a Guid on the `orgUser` when it's saved to the database. However, the sut uses 2 different repository methods, and we were only mocking one of them. This mocks the other to ensure a Guid is always set.